tests/i18n/tests.py

```diff
@@ -1658,132 +1578,6 @@ def test_fallback_language_code(self):
         self.assertEqual(li['code'], 'zh-hans')
 
 
-class MultipleLocaleActivationTests(SimpleTestCase):
```

Comment:
I'm not sure if duplicating this class 3 times is the best idea. At least, a common base class could be created. Maybe it's easiest to put trans/blocktrans/_ together. Or all "_" tests could go in a file rather than split between test_multiple_locales.py and test_string_literals.py.